components/atoms/Banner.js

```diff
@@ -9,9 +9,13 @@ export const Banner = ({ siteTitle, headline }) => {
   return (
     <div title="Banner image." className="bg-banner-img py-8 md:px-8">
       <div className="md:container w-full md:w-3/4 xl:w-1/2 bg-dk-blue bg-opacity-90 text-white p-4">
-        <h1 className="text-4xl pt-4 pb-2 break-words">{siteTitle}</h1>
+        <h1 className="font-display font-bold text-h1 pt-4 pb-2 break-words">
+          {siteTitle}
+        </h1>
         <hr className="border-2 border-hr-red-bar bg-hr-red-bar bg-opacity-90 border-opacity-90 w-3/4" />
-        <p className="text-xl py-4 break-words">{headline}</p>
+        <p className="font-body font-bold text-p py-4 break-words">
+          {headline}
+        </p>
```

Comment:
Is there a way to make the `h1`s and the `p`s take a default class? Adding a class to every single paragraph and header is something we can just avoid if we can set a global default. I saw you did something like this with your Next personal project, right?

Reply:
Thank you this is a great call and you are totally correct. I will implement that now.